src/common/eeprom.h

```diff
@@ -35,6 +65,10 @@ extern void eeprom_set_var(uint8_t id, variant8_t var);
 // fill range 0x0000..0x0800 with 0xff
 extern void eeprom_clear(void);
 
+extern void eeprom_get_hostname(char *dest);
```

Comment:
Maybe add documentation to this method mentioning that `dest` is expected to be of some size. Or except a `dest_len` parameter...

Reply:
@dragomirecky will be solved afterwards with eeprom refactoring